modules/aerodyn/src/AirfoilInfo_Types.f90

```diff
@@ -68,7 +68,7 @@ MODULE AirfoilInfo_Types
     REAL(ReKi)  :: k2      !< airfoil parameter in the x_cp_hat curve best-fit [ignored if UAMod<>1] [-]
     REAL(ReKi)  :: k3      !< airfoil parameter in the x_cp_hat curve best-fit [ignored if UAMod<>1] [-]
     REAL(ReKi)  :: k1_hat      !< Constant in the expression of Cc due to leading edge vortex effects.  [ignored if UAMod<>1] [-]
-    REAL(ReKi)  :: x_cp_bar      !< Constant in the expression of \hat(x)_cp^v [ignored if UAMod<>1, default = 0.2] [-]
+    REAL(ReKi)  :: x_cp_bar      !< Constant in the expression of hat(x)_cp^v [ignored if UAMod<>1, default = 0.2] [-]
```

Comment:
Since we use `\` as an escape character now, would we need `\\` to keep the `\` in the comment?

I'm not convinced it is worth bothering with here since we really don't use look at the `_Types.f90` files all that often.

Reply:
I don't think we count on doxygen picking up the _Types.f90 files correctly, so I don't think we need to fix it.